extensions/git/src/util.ts

```diff
@@ -149,6 +149,16 @@ export async function mkdirp(path: string, mode?: number): Promise<boolean> {
 	return true;
 }
 
+export async function createOrAppendFile(path: string, data: string, options?: any): Promise<boolean> {
```

Comment:
Opening the file is the right thing to do. I want to user to be aware of what is happening. We should show him what is being done behind the scenes here. If the user has auto-save enabled, the file should be automatically saved, right @bpasero?